src/platform-integration/web/faq.md

```diff
@@ -100,17 +100,65 @@ although no such support is built in.
 
 ### How do I embed a Flutter web app in a web page?
 
-You can embed a Flutter web app,
-as you would embed other content,
-in an [`iframe`][] tag of an HTML file.
-In the following example, replace "URL"
-with the location of your hosted HTML page:
+You have two embed options: embed in an `iframe` and embed in a `div` container.
```

Comment:
All this FAQ entry needs to do is to link to this page:

* As I mentioned before, this FAQ entry should link to this:

* https://docs.flutter.dev/deployment/web#embedding-a-flutter-app-into-an-html-page

That way we don't have to maintain two different copies of text on how to embed a flutter web app into an HTML page.